spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -95,7 +96,7 @@ def assert_specific_container_node
     expect(@containernode.ext_management_system).to eq(@ems)
   end
 
-  def assert_specific_container_service
+  def assert_specific_container_services
```

Comment:
why is the method renamed? (while there aren't other changes in the method body)